third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -535,6 +535,8 @@ function isInstanceOfNative(x, target) {
   const name = target?.name;
   return name &&
     x?.constructor?.toString()?.includes('() { [native code] }') &&
+    // fix https://linear.app/replay/issue/RUN-1067/chromium-css-style-object-preview-errors-out
```

Comment:
This comment isn't necessary, git blame can be used to recover this information and comments should generally be immediately descriptive / informative.

Reply:
Ok